.github/workflows/schedule-release.yml

```diff
@@ -0,0 +1,216 @@
+name: "Pine Continuous Deployment"
+
+on:
+  ## We will uncomment this when we settle on a deployment schedule
```

Comment:
Do we want to set this schedule prior to merging?

Reply:
If that's what we want. This could also be run manually like it is currently today.